pkg/app/piped/cmd/piped/piped.go

```diff
@@ -126,7 +126,11 @@ func NewCommand() *cobra.Command {
 }
 
 func (p *piped) run(ctx context.Context, input cli.Input) (runErr error) {
-	group, ctx := errgroup.WithContext(ctx)
+	// Make a cancellable context.
+	cCtx, cancel := context.WithCancel(ctx)
+	defer cancel()
+
+	group, ctx := errgroup.WithContext(cCtx)
```

Comment:
Why do we need to create a new context with new name?

```suggestion
	ctx, cancel := context.WithCancel(ctx)
	defer cancel()

	group, ctx := errgroup.WithContext(ctx)
```

Reply:
nice catch 👍